examples/chart/teleport-cluster/charts/teleport-operator/templates/resources.teleport.dev_opensshserversv2.yaml

```diff
@@ -0,0 +1,248 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: teleportopensshserversv2.resources.teleport.dev
+spec:
+  group: resources.teleport.dev
+  names:
+    kind: TeleportOpenSSHServerV2
+    listKind: TeleportOpenSSHServerV2List
+    plural: teleportopensshserversv2
+    shortNames:
+    - opensshserverv2
+    - opensshserversv2
+    singular: teleportopensshserverv2
+  scope: Namespaced
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        description: OpenSSHServerV2 is the Schema for the opensshserversv2 API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: OpenSSHServer resource definition v2 from Teleport
+            properties:
+              addr:
```

Comment:
should we make this property mandatory or should we leave it to teleport?
This is a situation where validation hooks would be great

Reply:
I think teleport should reject an invalid value and we should keep the CRD validation to the minimum: enforcing a sane data structure.